templates/course_homepage/course_homepage.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+     <meta charset="utf-8">
+      <title>NTU Course </title>
+      <link rel="stylesheet" href="course_homepage.css">
+      <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+   </head>
+
+   <!--大標-->
+   <body>
+      <div class="content">
+        <span class="text01">Course Helper </span>
+          <!--<a href="#">台大課程網 關鍵字篩選系統</a>
+          -->
+        </div>
+      <div class="explain">
+            <span>▍請輸入欲搜尋的課程關鍵字 ▍</span>
+            </div>
+     <div class="section-top"></div>
+    <!--搜尋按鈕---> 
+       <div class="search_box" method="GET">
+       <input type="text" name="" class="search_text" placeholder="" >
+       <!---<input class="search_button" href="#" type="submit" value="">-->
+       <a class="search_button" href="#">
+       <i class="fas fa-search"></i>
+       </a>
+       </div>
+    <!--Other Links-->
+       <div class="links">
+        <button class="btn btn1" >台大課程網</button>
```

Comment:
You could refer to [BEM](https://juejin.im/post/5b925e616fb9a05cdd2ce70d) for better class naming conventions.